src/System.Private.CoreLib/src/Internal/Runtime/Augments/RuntimeAugments.cs

```diff
@@ -158,6 +164,17 @@ public static unsafe Array NewMultiDimArray(RuntimeTypeHandle typeHandleForArray
             return Array.NewMultiDimArray(typeHandleForArrayType.ToEETypePtr(), pLengths, lengths.Length);
         }
 
+        public static void StelemRef(Array array, int index, object obj)
+        {
+            TypeCast.StelemRef(array, index, obj);
```

Comment:
I suspect that this won't compile in the closed source ProjectN build (the Runtime,Base is not part of CoreLib in that build). @MichalStrehovsky Do you agree?

I think it would be easier to delete this method; and replace it with `Unsafe.As<Object[]>(array)[index] = obj;`.

Reply:
Yeah, it wouldn't compile. Unsafe.As sounds good!